src/repository/screens/repository.screen.js

```diff
@@ -158,168 +162,212 @@ class Repository extends Component {
     return (
       <ViewContainer>
         {loader}
-
-        <ParallaxScroll
-          renderContent={() => {
-            if (isPendingRepository && !initalRepository) {
-              return <LoadingRepositoryProfile />;
-            }
-
-            return (
-              <RepositoryProfile
-                repository={isPendingRepository ? initalRepository : repository}
-                starred={
-                  isPendingRepository || isPendingCheckStarred ? false : starred
-                }
-                navigation={navigation}
-              />
-            );
-          }}
-          stickyTitle={repository.name}
-          showMenu={!isPendingRepository && !isPendingCheckStarred}
-          menuAction={() => this.showMenuActionSheet()}
-          navigation={navigation}
-          navigateBack
+        <ScrollView
+          refreshControl={
+            <RefreshControl
+              refreshing={isPendingRepository}
+              onRefresh={this.fetchRepoInfo}
+            />
+          }
         >
-          {initalRepository &&
-            !initalRepository.owner &&
-            isPendingRepository &&
-            <SectionList title="OWNER">
-              <LoadingUserListItem />
-            </SectionList>}
+          <ParallaxScroll
+            renderContent={() => {
```

Comment:
Please take this conditional outside of the `renderContent` prop, it should be performed outside of it.